src/app/dashboard/dashboard.component.ts

```diff
@@ -96,9 +99,18 @@ export class DashboardComponent implements OnInit {
     this.submissionsService.getSubmissions(findDocuments({
       'user.name': this.userService.get().name,
       type: 'survey',
-       status: 'pending'
+      status: 'pending'
     })).subscribe((surveys) => {
       this.surveys = surveys;
     });
   }
+
+  getExams() {

```

Comment:
This is a near duplicate of the `getSurveys()` function, and it is not querying for exams.  You should make a couple of changes here:

1. The `type` in the query needs to be 'exams' when querying for exams.  `status` should be `complete` because this is the status for an exam that is ready to be graded.
2. Please try to refactor `getExams()` and `getSubmissions()` into one function that is called once for exams and once for surveys in `ngOnInit()`.  Note that surveys should only query for submissions that match the username